src/web/core/src/main/resources/GeoServerApplication.properties

```diff
@@ -269,11 +269,11 @@ GeoServerHomePage.description         = Welcome
 GeoServerErrorPage.title              = Oops, something went wrong...
 GeoServerErrorPage.description        = Sorry, something unexpected happened on the server.
 GeoServerErrorPage.loginRequired      = If you were logged in, you could see an error report.
-GeoServerErrorPage.preamble           = Here's an error report you can include in a
-GeoServerErrorPage.jiraName           = JIRA bug report
+GeoServerErrorPage.userListLink       = http://geoserver.org/comm/
+GeoServerErrorPage.userListText       = Please follow this link, subscribe to the user list, and report this issue. Include the stack trace below and the steps to reproduce the problem.
```

Comment:
Perhaps the infamous "click here" to make it even more blindingly obvious. And perhaps a more general mention of help as it might be their problem not a problem with GeoServer? Something like: "Click here to subscribe to the GeoServer users mailing list. When asking for help, please include the stack trace below and the steps to reproduce the problem."